From 7df8feb9043d3e01d0781feb2ab5e986537bd5d2 Mon Sep 17 00:00:00 2001
From: Lennart Wieboldt <lennart.1997@gmx.de>
Date: Wed, 15 Apr 2015 19:48:48 +0200
Subject: [PATCH 50/95]  Enable -Os back to audioflinger & ID3

Aggressive -O2, -O3 optimizations on GCC 4.9 causes a seg-fault on playing some music tracks.

So enable -Os back to ID3.
Thanks @arter97
---
 media/libstagefright/id3/Android.mk | 2 +-
 services/audioflinger/Android.mk    | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/media/libstagefright/id3/Android.mk b/media/libstagefright/id3/Android.mk
index 2194c38..b72a19e 100644
--- a/media/libstagefright/id3/Android.mk
+++ b/media/libstagefright/id3/Android.mk
@@ -4,7 +4,7 @@ include $(CLEAR_VARS)
 LOCAL_SRC_FILES := \
 	ID3.cpp
 
-LOCAL_CFLAGS += -Werror
+LOCAL_CFLAGS += -Werror -Os
 
 LOCAL_MODULE := libstagefright_id3
 
diff --git a/services/audioflinger/Android.mk b/services/audioflinger/Android.mk
index 80179c1..1f52afa 100644
--- a/services/audioflinger/Android.mk
+++ b/services/audioflinger/Android.mk
@@ -70,6 +70,8 @@ LOCAL_STATIC_LIBRARIES := \
     libcpustats \
     libmedia_helper
 
+LOCAL_CFLAGS += -Os
+    
 LOCAL_MODULE:= libaudioflinger
 LOCAL_32_BIT_ONLY := true
 
-- 
2.5.0

